frontend/src/lib/components/frontPage/FrontPage.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  import {_} from 'svelte-i18n';
+  import {page} from '$app/stores';
+  import {Page} from '$lib/components/shared/page';
+  import {AppLogo} from '$lib/components/appLogo';
+  import {AppNav} from '$lib/components/appNav';
+  import {HelpIcon, MenuIcon} from '$lib/components/icons';
+  import type {FrontPageProps} from './FrontPage.type';
+
+  type $$Props = FrontPageProps;
+
+  export let title: $$Props['title'];
+  export let hgroupId: $$Props['hgroupId'] = 'mainHgroup';
+
+  // Merge default values to $$restProps
+  $$restProps.headerClass = `!absolute w-full bg-transparent ${$$restProps.headerClass ?? ''}`;
+  $$restProps.mainWrapperClass = `!p-0 ${$$restProps.mainWrapperClass ?? ''}`;
+  $$restProps.navLabel ??= $_('header.navLabel');
+</script>
+
+<Page {title} {...$$restProps}>
+  <!-- Header -->
+  <svelte:fragment slot="navOpen">
+    <MenuIcon />
+    <AppLogo />
```

Comment:
Yeah, that would be the standard functionality. However, in this case it might be confusing to the user whether the menu button is its separate button or part of the logo. As opening the menu would most likely be the more useful interaction here, I'd keep it this way. The home page can be anyway reached via a second click after the menu is open.